core/src/main/java/org/kiwix/kiwixmobile/core/data/local/entity/BookmarksSpec.kt

```diff
@@ -15,18 +15,26 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
-package org.kiwix.kiwixmobile.core.data.local.entity;
+package org.kiwix.kiwixmobile.core.data.local.entity
 
-import com.yahoo.squidb.annotations.TableModelSpec;
-import com.yahoo.squidb.annotations.tables.constraints.NotNull;
+import com.yahoo.squidb.annotations.TableModelSpec
+import com.yahoo.squidb.annotations.tables.constraints.NotNull
 
 /**
  * Squidb spec for saved bookmarks.
  */
+@Suppress("VariableNaming")
```

Comment:
Hi @gouri-panda ,

We have 
``` 
@NotNull
  lateinit var ZimId: String

@NotNull
  lateinit var ZimName: String
  ```

and these varibles start with caps letter so that's why we need this.